src/services/DappRadarService.ts

```diff
@@ -146,10 +200,26 @@ export class DappRadarService {
         // In some cases dapp name in dapp staking and in dapp radar are not exactly the same, so idea to check if
         // name or dapp url match. If both are different most likely the dapp will need to update name or url in dapp staking.
         return dapps.find(
-            (x) => x.name.toLowerCase() === dappName.toLowerCase() || x.website.toLowerCase() === dappUrl.toLowerCase(),
+            (x) =>
+                x.name.toLowerCase() === dappName.toLowerCase() ||
+                this.getDomain(x.website.toLowerCase()) === this.getDomain(dappUrl.toLowerCase()),
         )?.dappId;
     }
 
+    private getDomain(url: string): string | null {
+        var prefix = /^https?:\/\//i;
```

Comment:
why don't you use `const` in this method?

Reply:
Fixed, thank you.